lib/wpcc/version.rb

```diff
@@ -1,8 +1,8 @@
 module Wpcc
   module Version
     MAJOR = 2
```

Comment:
Being a Ruby upgrade I believe the general consensus is to bump the Major version

Reply:
I followed [this pr](https://github.com/moneyadviceservice/savings_calculator/pull/32/files). And I'm not sure I agree that a ruby upgrade should necessarily be a `Major` bump. Maybe if it was a bigger upgrade, but I think the size of the bump may need to be on a case by case basis. Happy to hear other thoughts.

Reply:
I started doing major bumps, now I go for minor or even patches as, once thought about what we're actually doing, there are 0 logic changes there. In theory a Major bump should be for breaking changes.